File: 백준/Bronze/10818. 최소， 최대/최소， 최대.cs
int cnt = int.Parse(Console.ReadLine());
int min = 1000000;
int max = -1000000;

string[] str = Console.ReadLine().Split();
int[] nums = new int[cnt];

for (int i = 0; i < cnt; i++)
{
    nums[i] = int.Parse(str[i]);
}

for (int i = 0; i < cnt; i++)
{
    if(max < nums[i]) max = nums[i];
    if(min > nums[i]) min = nums[i];
}

Console.WriteLine(min + " " + max);

File: 백준/Bronze/10818. 최소， 최대/최소， 최대.py
cnt = int(input())
nums = list(map(int, input().split()))

min_val = 1000000
max_val = -1000000

for num in nums:
    if num > max_val:
        max_val = num
    if num < min_val:
        min_val = num

print(min_val, max_val)
